### PR TITLE
fix(api): remove duplicate New Relic import in bootstrap file

### PR DESCRIPTION
**Pull request by merrcury**
### Screenshots

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

**Review by SokratisVidros — APPROVED** [review]

File: apps/api/src/bootstrap.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
+import 'newrelic';
```
Comment (SokratisVidros) [inline apps/api/src/bootstrap.ts]: You can also move the import to `./instrument` and import `./instrument` as the first thing.

**Comment by SokratisVidros** [comment]
@merrcury Please apply the same change to all bootstrap files for all services.

File: apps/api/src/bootstrap.ts (hunk 1 above)
Comment (merrcury) [reply apps/api/src/bootstrap.ts]: Wouldn't it create problem with import './config/env.config'? As instrument does make use of it?

Comment (merrcury) [reply apps/api/src/bootstrap.ts]: Shall I move that inside instrument too?

Comment (SokratisVidros) [reply apps/api/src/bootstrap.ts]: Yes

**Pull request merged**